### PR TITLE
refactor(rxCapitalize): FRMW-371 move rxCapitalize to quark

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-371

- [x] Dev LGTM
- [x] Dev LGTM
- [x] Design LGTM

---
Refactor `rxCapitalize` filter to a quark.

### CODE REVIEW

**Comment by 100stacks** [comment]
### New rxCapitalize Page

screen shot 2015-12-10 at 1 42 46 pm

**Comment by 100stacks** [comment]
## Quark API Page with rxCapitalize Filter 

screen shot 2015-12-10 at 1 44 41 pm

screen shot 2015-12-10 at 1 45 00 pm

File: src/components/tabs/docs/tabs.midway.js [hunk 1]
```diff
@@ -83,11 +83,11 @@ describe('tabs', function () {
     describe('main tabs', function () {
 
         before(function () {
-            demoPage.go('#/components/rxCapitalize');
+            demoPage.go('#/quarks/rxCapitalize');
```
Comment (100stacks) [inline src/components/tabs/docs/tabs.midway.js]: Update to new `rxCapitalize` location.

File: src/components/tabs/docs/tabs.midway.js [hunk 2]
```diff
@@ -83,11 +83,11 @@ describe('tabs', function () {
...
         });
 
         it('should find the only tabs on the page', function () {
-            expect(tabsPage.main.names).to.eventually.eql(['Demo', 'Markup', 'JavaScript', 'Protractor']);
+            expect(tabsPage.main.names).to.eventually.eql(['Demo', 'Markup', 'JavaScript']);
```
Comment (100stacks) [inline src/components/tabs/docs/tabs.midway.js]: New demo page has three default tabs - **Demo**, **Markup**, and **JavaScript**.

**Comment by glynnis** [comment]
LGTM

**Comment by CITguy** [comment]
LGTM

**Comment by parlarjb** [comment]
LGTM

**Pull request merged**